v4/source/channels.yaml

```diff
@@ -80,6 +80,36 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  /channels/group:
+    post:
+      tags:
+        - channels
+      summary: Create a group message channel
+      description: |
+        Create a new group message channel to group of users.
```

Comment:
Can we add a note along the lines of "If the logged in user's id is not included in the list, it will be appended to the end."?